packages/calcite-components/src/components/list/list.scss

```diff
@@ -13,9 +13,7 @@
   flex
   w-full
   flex-col
-  bg-transparent
-  relative
-  z-default;
+  bg-transparent;
```

Comment:
We might still need to bump the z-index of the `filter-enabled` UI elements - with this change it is behind the focused list element:

https://github.com/Esri/calcite-design-system/assets/4733155/b57ad989-531a-410b-868d-a20c2b5b1109


https://github.com/Esri/calcite-design-system/assets/4733155/07f6b932-7e9e-4cc8-ba90-e7001f094b53

Reply:
Nice catch @macandcheese. I removed the z-index on the active element. Not sure why we had it there. @jcfranco any idea?

Reply:
I don't recall. Nice cleanup though! 🧼